src/svgcontext.ts

```diff
@@ -230,6 +230,14 @@ export class SVGContext extends RenderContext {
     this.parent = this.groups[this.groups.length - 1];
   }
 
+  openRotation(angleDegrees: number, x: number, y: number) {
+    this.openGroup().setAttribute('transform', `translate(${x},${y}) rotate(${angleDegrees}) translate(-${x},-${y})`);
```

Comment:
I think in CSS, you are supposed to specify the units like 90deg, but does it default to degrees if you omit the unit?